src/buildzqfield.js

```diff
@@ -92,7 +92,7 @@ if (runningAsScript) {
     const q = bigInt(argv.q);
 
     const asmFileName =  (argv.oa) ? argv.oa : argv.name.toLowerCase() + ".asm";
-    const no_adx =  (argv.no_adx) ? argv.no_adx : false;
+    const no_adx =  (argv.no_adx) ? argv.no_adx : null;
```

Comment:
I think this should be a boolean flag

```suggestion
    const no_adx =  (argv.no_adx) ? argv.no_adx : false;
```

Reply:
no because I need to know if flag was not provided at all, if it wasnt we still need to try for adx support to use it

https://github.com/zkbitcoin/ffiasm/blob/support-no-adx/src/buildzqfield.js#L68

Reply:
Oh I see. then isn't `undefined` value is more proper for this purpose?

Reply:
agree will test with undefined